src/lib/backend.ai-client-node.ts

```diff
@@ -2633,6 +2633,12 @@ class Maintenance {
     this.urlPrefix = '/resource';
   }
 
+  attach_background_task(task_id: string) {
```

Comment:
@mickowale We need a comment section above this method. (It is referenced by TypeScript to check the parameter type)